.github/workflows/build_and_push_images.yml

```diff
@@ -0,0 +1,53 @@
+name: "Build and Push Images"
+
+on:
+  push:
+    paths:
+      - "app/**"
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    strategy:
+      matrix:
+        tag:
+          - frontend
+          - backend1
+          - backend2
```

Comment:
we are not publishing the backend3?

Reply:
I could but it is not really used through out the tutorial, but if people like to play around with things and have it eventually, I can put it out there as well.

Reply:
Since everything is already in place, we can publish it. At least there will be some CI to make sure it actually builds.

Reply:
Let me add it then, actually, what I do before that I rename them to "backend-python", "backend-java" ... to make it easier to distinguish them :)

Reply:
Added backend3 as well

Reply:
Aaaand I removed it again, didn't build properly...